recipe/meta.yaml

```diff
@@ -44,10 +44,6 @@ test:
 
   commands:
     - pip check
-    # Disable newer Arrow strings with dask-image.
-    # xref: https://github.com/dask/dask-image/issues/335
-    - export DASK_DATAFRAME__CONVERT_STRING="False"  # [unix]
-    - set "DASK_DATAFRAME__CONVERT_STRING=False"     # [win]
     - pytest
```

Comment:
This was here to workaround issues like ( https://github.com/dask/dask-image/issues/355 ) and ( https://github.com/dask/dask-image/issues/335 ), but it appears no longer to be needed. So dropped it